src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -247,11 +247,11 @@ public DraftApprenticeship AddDraftApprenticeship(DraftApprenticeshipDetails dra
             return draftApprenticeship;
         }
 
-        public virtual void Approve(Party modifyingParty, string message, UserInfo userInfo, DateTime now)
+        public virtual void Approve(Party modifyingParty, string message, UserInfo userInfo, DateTime now, bool apprenticeEmailRequired)
```

Comment:
can we give this a default value of false? this may help in not changing some of the tests.

Reply:
done